### DISCUSSION TITLE
values stored as x10 or x100 original value in influxdb

### DISCUSSION THREAD

**Discussion by Innocalente** [johanmeijer/grott#338] [Q&A] [answered]
First off all a big thanks to @johanmeijer; thank you very much for Grott! just what I was looking for.

Maybe somebody can help me with the following:
- I have Grott working as a server, storing the data in influxdb.
- Grott logg shows it recieves the correct data from the inverter (for example the value 21.4)
* However, the log also shows that the value 214 is stored in influxdb (without the comma, thus x10 some values x100) 

What am I missing here? What am I doing wrong?

Any help would be greatly appreciated

**Comment by johanmeijer** [comment, marked as answer]
You doing nothing wrong. Grott store the raw values (as being sent by the inverter) into influxDB. The units are sometimes different then expected so for power it is 0.1W (1000 is 100 watt), for Voltage is 0.1V, for frequency 0.01hz (50 Herz is 5000), for Energy it 0.1kWh and temp 0.1C. A strange one is the working time that is 0.5 Sec. 

In the logging of Grott I format this values to display a normal readable field (is part of the layout definition). I choose to export the unformatted format to influxDB and Mqtt. So still have to do some math to get readable values in your dasboards.

**Reply by Innocalente** [reply]
Thanks! do you have a monero address ?

**Comment by johanmeijer** [comment]
Your welcome. No I don't have a monero address.....

**Reply by Innocalente** [reply]
I should have explained a bit better; I sincerely appreciate your work on grott. Therefore I would like to send you 'a cup of coffee/pizza/beer'. Is there any other way I can send you some crypto?

**Reply by johanmeijer** [reply]
Hmm no, I lost (hopefully temporary, it should be somewhere in the paper mesh) the key of my wallet :( . You can send me some real money via PayPal (donate button in the read.me).

**Reply by rany2** [reply]
> send me some real money via PayPal

I don't know why but that made me laugh :laughing:

**Reply by johanmeijer** [reply]
You mean it is all virtual?

**Reply by rany2** [reply]
No, it's just that what you said is all very true but I felt like you said it in a funny way... at any rate I recommend you make a Monero wallet regardless and just trade it for real money with some exchange; that's what many people do. Most people that want to donate Monero would never use PayPal anyway, so you're missing out.
